backend/entities/Views/Property.cs

```diff
@@ -24,6 +24,13 @@ public class Property
         /// </summary>
         public string ProjectNumber { get; set; }
 
+
+        /// <summary>
+        /// get/set - The RAEG/SPP project status
+        /// </summary>
+        [NotMapped]
+        public string ProjectStatus { get; set; }
```

Comment:
Again only include in the API models, not in the DAL.  Additionally it doesn't make sense at this level, as properties can be associated to many projects and thus have many status (although only one that is active at any given point in time).